tests/chart_tests/test_sts_volume_claim_annotations.py

```diff
@@ -0,0 +1,138 @@
+import pytest
+
+from tests import supported_k8s_versions
+from tests.chart_tests.helm_template_generator import render_chart
+
+
+@pytest.mark.parametrize(
+    "kube_version",
+    supported_k8s_versions,
+)
+class TestStatefulSetsAnnotations:
+    def test_sts_with_no_annotations(self, kube_version):
+        """Test all sts for the volume claim templates annotations"""
+        docs = render_chart(
+            kube_version=kube_version,
+            values={},
+            show_only=[
+                "charts/elasticsearch/templates/master/es-master-statefulset.yaml",
+                "charts/elasticsearch/templates/data/es-data-statefulset.yaml",
+                "charts/alertmanager/templates/alertmanager-statefulset.yaml",
+                "charts/astronomer/templates/registry/registry-statefulset.yaml",
+                "charts/prometheus/templates/prometheus-statefulset.yaml",
+                "charts/stan/templates/statefulset.yaml",
+            ],
```

Comment:
Can we replace this with something that auto-discovers sts? We could either build this list by walking the filesystem, or we could render the full chart and filter it for sts, which is how we've done this elsewhere. <https://github.com/astronomer/astronomer/blob/d46aacf2ec/tests/chart_tests/test_default_chart.py#L10-L18>

Reply:
OR change the docstring to not say "Test **all** sts"